src/std/main.ab

```diff
@@ -111,6 +111,28 @@ pub fun sum(list: [Num]): Num {
     return unsafe $echo "{list}" | awk '\{s=0; for (i=1; i<=NF; i++) s+=\$i; print s}'$ as Num
 }
 
+pub fun array_value(array, index) {
+    return array[index]
+}
+
+pub fun array_index(array, value): Num {
+    loop index, element in array {
+        if(value as Text == element as Text) {
+            return index
+        }
+    }
+}
+
+pub fun array_search(array, value): [Num] {
```

Comment:
Wondering if it is better something like `in_array` or `array_search_by_value`.
Also looking at `array_index` maybe this function can use it.

Reply:
I added the in_array function

Reply:
i can rename ``array_index`` to ``array_first_index``

Reply:
let's see the others what think about it.
we are just creating a new language so define the naming of the functions is important.

Reply:
what about calling it `array_find_first`? or just return an array of all found indexes if there are duplicates of the same element